e2e/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id("java")
+}
+
+group = "tbdex"
+version = "0.0.0"
+
+repositories {
+    mavenCentral()
+    maven {
+      url = uri("https://repo.danubetech.com/repository/maven-public")
+    }
+    maven {
+      url = uri("https://jitpack.io")
+    }
+}
+
+dependencies {
+  implementation(project(":protocol"))
+  implementation(project(":httpclient"))
+  implementation("me.lessis:typeid:0.0.2")
+  implementation("com.github.TBD54566975:web5-kt:0.0.0-beta")
+  testImplementation(platform("org.junit:junit-bom:5.9.1"))
+  testImplementation("org.junit.jupiter:junit-jupiter")
+}
```

Comment:
I think we're also using `testImplementation(kotlin("test"))` in all the other packages